petclinic/actors/vets/src/db.rs

```diff
@@ -1,7 +1,7 @@
 use super::Db;
 use serde::{Deserialize, Serialize};
 use wasmbus_rpc::actor::prelude::*;
-use wasmcloud_interface_sqldb::{FetchResult, SqlDb, SqlDbError, minicbor};
+use wasmcloud_interface_sqldb::{minicbor, FetchResult, SqlDb, SqlDbError};
```

Comment:
This is an enhancement for another time, but rather than re-exporting the entire crate, I'd prefer it if we just exported the appropriate encoder and decoder, which then doesn't bleed the implementation detail (minicbor) into the interface.

Reply:
I agree with the idea of making an abstraction layer. It's more than just an encode and decode functions - it would be at least 20 functions (one for each data type, for both encode and deceode), but it would provide some insulation which is a good thing. I'll make a separate issue for that.